.tina/collections/events.tsx

```diff
@@ -41,6 +41,25 @@ export const eventsSchema: Collection = {
       name: "showTestimonials",
       label: "Show Testimonials",
     },
+    {
+      type: "object",
+      label: "Testimonial Categories",
+      name: "testimonialCategories",
+      ui: {
+        itemProps(item) {
+          return { label: item.testimonialCategory }
```

Comment:
Item can be null here, can you add a fallback label?

Reply:
Done - added a fallback label as "Select your testimonial category"